libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -208,6 +212,9 @@ constructor(
         fasterRouteController = FasterRouteController(directionsSession, tripSession, logger)
         routeRefreshController = RouteRefreshController(directionsSession, tripSession, logger)
         routeRefreshController.start()
+
+        arrivalProgressObserver = ArrivalProgressObserver(tripSession)
+        attachArrivalController(AutoArrivalController())
```

Comment:
See suggestion. This is not needed as `ArrivalProgressObserver` creates `AutoArrivalController()` by default.

```suggestion
```

Reply:
still need to initialize the `arrivalProgressObserver`

Reply:
```suggestion
        attachArrivalController()
```